src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1440,7 +1454,7 @@ await PreviewBuildIntegratedProjectActionsAsync(buildIntegratedProject, actions,
 
                 foreach (var package in packageIdentities)
                 {
-                    var packageDependencyInfo = await dependencyInfoResource.ResolvePackage(package, nuGetFramework, CancellationToken.None);
+                    var packageDependencyInfo = await dependencyInfoResource.ResolvePackage(package, nuGetFramework, Logging.NullLogger.Instance, CancellationToken.None);
```

Comment:
I've opened https://github.com/NuGet/Home/issues/2035 to track this. It will be a large work item to determine where this output should be displayed and to verify that it is user friendly for the scenario.